spyder/widgets/editor.py

```diff
@@ -0,0 +1,551 @@
+# -*- coding: utf-8 -*-

```

Comment:
Please rename this file to `simplecodeeditor.py`. `editor.py` is very vague and could lead to confusions later.

Reply:
Fixed.